engine/src/physics/plugin.cpp

```diff
@@ -89,6 +114,7 @@ void cubos::engine::physicsPlugin(Cubos& cubos)
                 cmds.add(ent, force);
                 cmds.add(ent, impulse);
                 cmds.add(ent, AccumulatedCorrection{});
+                cmds.add(ent, PhysicsMaterial{});
```

Comment:
Maybe you should add a `PhysicsMaterial material` field to the `PhysicsBundle` component, and then add it here?
That way the user can configure the material more easily.